### ISSUE TITLE
Discovery of precision errors

### ISSUE THREAD

**Issue by szabo137** [QEDjl-project/QEDcore.jl#22]
The tests should be redesigned in a way, that precision errors are discovered. This would open up the possibility of implementing more precise algorithms for accuracy-sensitive tasks like `isonshell`. 

See e.g. comment posted by @SimeonEhrig in https://github.com/QEDjl-project/QEDbase.jl/pull/50#discussion_r1426464091_

**Comment by SimeonEhrig** [comment]
> I believe we could use https://github.com/AntonReinhard/PrecisionCarriers.jl to automatically test some precision metrics at some point.

I like how you suggest your self-written package like it would be package from some random, unknown guy, who had a clever idea 😂

**Comment by SimeonEhrig** [comment]
No, should hide that you wrote the package. Otherwise you will be catch by a big tech company 😂
